Configuration/StandardSequences/python/RawToDigi_cff.py

```diff
@@ -104,7 +103,7 @@
 
 # GEM settings
 _gem_RawToDigiTask = RawToDigiTask.copy()
-_gem_RawToDigiTask.add(muonGEMDigis)
+_gem_RawToDigiTask.add(muonGEMDigiTask)
```

Comment:
why are these modules not added to L1Reco step/sequence instead?
IMHO, RAW2DIGI should be restricted to format conversion and not run reconstruction algorithms which can change with release.

Reply:
@slava77 Sure. please show me the file that needs to be modified instead.

Reply:
L1Reco step is defined based on the contents of Configuration/StandardSequences/python/L1Reco_cff.py and by default is using `L1Reco` sequence